postcode_lookup/template_sorter.py

```diff
@@ -102,7 +103,7 @@ def weight(self):
             poll_date - datetime.timedelta(days=days_before_poll)
         ) < self.current_date:
             return -6000
-        return 1001
+        return 1003
```

Comment:
I found the process of re-jigging the weight values a bit unintuitive and as we add more components this is going to get more fiddly. I don't think I have a concrete suggestion right now, but I'd like to have a think about how we can do this same concept better.